README.md

```diff
@@ -145,8 +145,10 @@ ostorlab vulnz describe -v <vuln-id>
 
 For the moment, we are currently focused on the CISA KEV Database and Google Tsunami.
 
-| CVE ID                                                  | Implemented  | Detail                                                  | Published Date |
-|:--------------------------------------------------------|:------------:|:--------------------------------------------------------|:--------------:|
+| CVE ID                                                  | Implemented | Detail                                                  | Published Date |
+|:--------------------------------------------------------|:-----------:|:--------------------------------------------------------|:--------------:|
+| CVE-2024-4885                                           |      ✅      | Official Nuclei template.                               |   2024-06-25   |
+| CVE-2024-37032                                          |      ✅      | Official Nuclei template.                               |   2024-05-31   |
```

Comment:
I only see one file and you didn't update the agent group

Reply:
I added one file, I had conflicts before.